src/js/containers/search/visualizations/geo/GeoVisualizationSectionContainer.jsx

```diff
@@ -332,6 +341,166 @@ export class GeoVisualizationSectionContainer extends React.Component {
         });
     }
 
+    updateMapScope() {
```

Comment:
you should be able to create a common function here a call it once for place of performance and once for the recipient location.  you can pass in a parameter for the filter type (ie. call updateMapScope("pop") and updatemapScope("recipientLocation") and then just use a variable for this.props.reduxFilters.selectedLocations.  for example 

`const selectedLocationByType = type === "pop" ? "selectedLocation" : "selectedRecipientLocations"`
then use
`this.props.reduxFilters[selectedLocationByType]`